.azure-pipelines/client.yml

```diff
@@ -0,0 +1,180 @@
+trigger:
+- psSdkJson6
+
+#Variables: 
+#  'source.scope' list of projects to build (Ex: SDKs\KeyVault\dataPlane)
+#  'build.useLatest' indicates whether or not to use build bootstrap files from the LTSBuildTools branch
+#  'sign.pattern' the pattern used to find assemblies for SN and AuthentiCode signing
+#  'system.debug' bool for debug tracing
+
+#TODO: sign.pattern needs better filtering (not KeyVault specific, exclude test assemblies).
+
+variables:
+  source.scope: 'SDKs\KeyVault\dataPlane'
```

Comment:
@kurtzeborn maybe I don't understand the way devOps pipeline works.
But any reason we are hardcoding scope here (currently hardcoded to keyVault data plane)
Does this mean you will have separate yml for each SDK?
Can the scope be parameterized in devOps?

Reply:
I'll take the variables values out.  I was using these wrong.  What I had would result in the value passed in from DevOps to be overwritten every time with this value.